pkg/utils/utils.go

```diff
@@ -534,12 +534,13 @@ func getVfInfo(pciAddr string, devices []*ghw.PCIDevice) sriovnetworkv1.VirtualF
 	return vf
 }
 
-func LoadKernelModule(name string) error {
-	glog.Infof("LoadKernelModule(): try to load kernel module %s", name)
-	cmd := exec.Command("/bin/sh", scriptsPath, name)
+func LoadKernelModule(name string, args []string) error {
```

Comment:
you can do a variadic declaration of args , this way if not provided it will evaluate to empty list, else you will have the args in the list

e.g
`LoadKernelModule(name string, args ...string)`

then you can call it with or without args

Reply:
good idea